tests/wpt/metadata/html/semantics/forms/constraints/form-validation-validity-stepMismatch.html.ini

```diff
@@ -1,101 +1,5 @@
 [form-validation-validity-stepMismatch.html]
   type: testharness
-  [[INPUT in DATETIME status\] The step attribute is not set]
-    expected: FAIL
-
-  [[INPUT in DATETIME status\] The value attibute is empty string]
-    expected: FAIL
-
-  [[INPUT in DATETIME status\] The value must match the step]
-    expected: FAIL
-
-  [[INPUT in DATETIME status\] The value must mismatch the step]
-    expected: FAIL
-
-  [[INPUT in DATE status\] The step attribute is not set]
-    expected: FAIL
-
-  [[INPUT in DATE status\] The value attibute is empty string]
-    expected: FAIL
-
-  [[INPUT in DATE status\] The value must match the step]
-    expected: FAIL
-
-  [[INPUT in DATE status\] The value must mismatch the step]
-    expected: FAIL
-
-  [[INPUT in MONTH status\] The step attribute is not set]
-    expected: FAIL
-
-  [[INPUT in MONTH status\] The value attibute is empty string]
-    expected: FAIL
-
-  [[INPUT in MONTH status\] The value must match the step]
-    expected: FAIL
-
-  [[INPUT in MONTH status\] The value must mismatch the step]
-    expected: FAIL
-
-  [[INPUT in WEEK status\] The step attribute is not set]
-    expected: FAIL
-
-  [[INPUT in WEEK status\] The value attibute is empty string]
-    expected: FAIL
-
-  [[INPUT in WEEK status\] The value must match the step]
-    expected: FAIL
-
-  [[INPUT in WEEK status\] The value must mismatch the step]
-    expected: FAIL
-
-  [[INPUT in TIME status\] The step attribute is not set]
-    expected: FAIL
-
-  [[INPUT in TIME status\] The value attibute is empty string]
-    expected: FAIL
-
-  [[INPUT in TIME status\] The value must match the step]
-    expected: FAIL
-
-  [[INPUT in TIME status\] The value must mismatch the step]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] The step attribute is not set]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] The value attibute is empty string]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] The value must match the step]
-    expected: FAIL
-
-  [[INPUT in DATETIME-LOCAL status\] The value must mismatch the step]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] The step attribute is not set]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] The value attibute is empty string]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] The value must match the step]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] The value must mismatch the step]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] The step attribute is not set and the value attribute is a floating number]
-    expected: FAIL
-
-  [[INPUT in NUMBER status\] The value attribute is empty string]
-    expected: FAIL
-
   [[INPUT in NUMBER status\] Step mismatch when step is a very small floating number and value is not its integral multiple]
     expected: FAIL
```

Comment:
This doesn't pass, because we parse numbers as floats, see https://github.com/whatwg/html/issues/5207.